docs/api.md

```diff
@@ -321,6 +322,133 @@ granted). You can pass the parameter `full_perms=true` to API calls to view the
 full permissions of objects in a format that mirrors the `set_permissions`
 parameter above.
 
+## Permission options
```

Comment:
Thanks for the PR. My only concern is going to be a bit of a chore to maintain and Im not sure how helpful it is as just a straight list. Hmm...

Reply:
We also dont use all of them in a meaningful way so again, Im not sure of the purpose of an exhaustive list.

Reply:
Hi shamoon,

i work on an ansible-collection for configure paperless-ngx with ansible. creation tags, corresponders and as well users and so far. The sense was to serve a list with all possible options.

Reply:
the values coming from the api from the paperless default user, if you think, it's useless, no worry